Deploy-CAM.ps1

```diff
@@ -3015,7 +3076,7 @@ else {
     }
     do {
         if ( -not $password ) {
-            $psw = Read-Host -AsSecureString "Please enter the password"
+            $psw = (Read-Host -AsSecureString "Please enter the password").Trim()
```

Comment:
Should we trim password? I thought spaces are allowed for password. I'm not sure if you can have leading or ending with spaces in password tho.

Reply:
Good point! And probably the RADIUS shared secret as well.